src/components/Tooltip/index.js

```diff
@@ -141,47 +142,50 @@ class Tooltip extends PureComponent {
         TooltipSense.deactivate();
     }
 
+    /**
+     * Returns true if children is a focusable component.
+     *
+     * @returns {Boolean}
+     */
+    isFocusable() {
+        const name = this.props.children.type.displayName;
+        const isPressableText = name === 'Text' && Boolean(this.props.children.props.onPress);
+        return isPressableText || ['TouchableOpacity', 'Pressable', 'TouchableWithoutFeedback'].includes(name);
+    }
```

Comment:
What about the new condition I posted above here https://github.com/Expensify/App/pull/16052#issuecomment-1478897250? Actually. I prefer the new condition because we don't need to maintain the component list (and we should add more component to the list beside touchable/pressable component, for example `TextInput` even though we don't have any text input wrapped with tooltip).

Reply:
`isFocusable` is only to decide the `focusable` of the children, not whether we should apply a View wrapper or not. So, there will be no restructure.

Reply:
@parasharrajat Bump on above. I will update the code once we agree with the `isFocusable` logic.

Reply:
The `isFocusable` function won't cause any re-render as there is no set state inside it. When the ref is ready (which should be on mount), `isFocusable` will return either true or false. The value just depends on ref.

Reply:
There won't be unnecessary re-render. When the ref is not ready, Tooltip will be rendered with `focusable` value of `undefined` (not false). When the ref is ready (mounted), Tooltip will now have `focusable` value of either `true` or `false`. `isFocusable` is just a derived value from tooltip `ref`. That's the same thing when you are trying to render something based on a value that changes, which in our case is the `ref`. With or without `isFocusable`, Tooltip will be re-rendered when the `ref` is ready.